lib/pdn/pdn.c

```diff
@@ -130,6 +130,8 @@ static void on_cgev(const char *notif)
 		{"ME PDN ACT",	 PDN_EVENT_ACTIVATED},	 /* +CGEV: ME PDN ACT <cid>[,<reason>] */
 		{"ME PDN DEACT", PDN_EVENT_DEACTIVATED}, /* +CGEV: ME PDN DEACT <cid> */
 		{"NW PDN DEACT", PDN_EVENT_DEACTIVATED}, /* +CGEV: NW PDN DEACT <cid> */
+		{"ME DETACH",	 PDN_EVENT_DETACHED},	 /* +CGEV: ME DETACH */
+		{"NW DETACH",	 PDN_EVENT_DETACHED},	 /* +CGEV: NW DETACH */
```

Comment:
Alternatively, we send the existing `_DEACTIVATED` message to all callbacks. Do callbacks need to know that all other PDNs are also deactivated? I understand this might be a slightly different situation under the hood, but does the application need that knowledge?

Reply:
We have users counting the number of ACTIVATED and DEACTIVATED events. In mfw 1.3.x we get both "ME PDN DEACT" and "ME DETACH" which will give inconsistency in the counting. This is why I added a new DETACHED.